test/builder_test.rb

```diff
@@ -1,8 +1,8 @@
 require 'test_helper'
-require 'elasticquery/queries/all'
+require 'elasticquery/builder'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.